src/installer/corehost/cli/test/nativehost/CMakeLists.txt

```diff
@@ -45,6 +45,10 @@ if(CLR_CMAKE_TARGET_WIN32)
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /DELAYLOAD:nethost.dll")
 endif()
 
+include(../../hostmisc/hostmisc.cmake)
+list(APPEND SOURCES ${HOSTMISC_SRC})
+list(APPEND HEADERS ${HOSTMISC_HDR})
```

Comment:
Can this go in `testexe.cmake` instead? I think pal/utils would probably be useful for any native test executables. I know `nativehost` is the only thing including `testexe.cmake` right now, but the intention is that `test_fx_ver` (and any future native test executables) will use it.

Reply:
No needed.